repositories.bzl

```diff
@@ -187,6 +186,7 @@ copy_file(
         strip_prefix = "async-profiler-4.1-macos/lib",
         urls = ["https://github.com/async-profiler/async-profiler/releases/download/v4.1/async-profiler-4.1-macos.zip"],
     )
+    return ctx.extension_metadata(reproducible = True)
```

Comment:
I remember having this break the build since we parse the lockfile to generate the dist info.